ext/data/recommended-dictionaries.json

```
@@ -1,13 +1,25 @@
 {
     "ja": {
         "terms": [
-            {"name": "Jitendex", "url": "https://github.com/stephenmk/stephenmk.github.io/releases/latest/download/jitendex-yomitan.zip"}
+            {
+                "name": "Jitendex",
+                "url": "https://github.com/stephenmk/stephenmk.github.io/releases/latest/download/jitendex-yomitan.zip",
+                "description": "An improved version of JMdict for Yomitan. It features better formatting and some other improvements, and is actively being improved by its author."
```

```suggestion
                "description": "A free and openly licensed Japanese-to-English dictionary with example sentences, usage notes, etymology notes, cross references, antonyms, definition notes."
```

Let's use a modified version of Jitendex.org's description